ubuntudesign/vanilla-brochure-theme
# Add .strip--image pattern

## Done

- Added ` .strip--image` pattern

## QA

- Check with example site by pulling down branch and running `gulp jekyll`
- Navigate to `http://127.0.0.1:4001/vanilla-brochure-theme/examples/patterns/strip/image/` 
- Compare to [design spec](https://raw.githubusercontent.com/ubuntudesign/vanilla-brochure-theme-design/master/Strip/strip.png)

# Related

Fixes #12
